src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/OrderDetails/OrderDetailsController.ts

```diff
@@ -58,7 +59,11 @@ module Orckestra.Composer {
                         this.Modal.cancelOrderModal.openModal(event);
                     },
                     cancelOrder(){
-                        self.orderService.cancelOrder(this.OrderNumber); 
+                        if(this.Loading) return;
+                        this.Loading = true;
+                        self.orderService.cancelOrder(this.OrderNumber).fail(() => {
```

Comment:
can be simplified L_UpdatingOrderFailed self.orderService.cancelOrder(this.OrderNumber).fail(() => this.Loading = false);